### PR TITLE
add team name to raid alert

### PR DESCRIPTION
**Pull request by ngovil21**
## Description
Add raid team name to raid alerts.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (would cause existing functionality to change)

## Motivation and Context

This gives more information on the current team controlling the raid. This information is useful because premiere balls are given based on the team controlling the raid. Gym team information is also extracted and saved earlier if gym notifications have been disabled.

## How Has This Been Tested?
This has been tested in a production environment with Discord notifications.

## Screenshots (if appropriate):
image

## Wiki Update

- [x] This change requires an update to the Wiki.
- [ ] This change does not require an update to the Wiki.

## Checklist
- [x] This change follows the code style of this project.
- [x] This change only changes what is necessary to the feature.

### CODE REVIEW

File: PokeAlarm/Manager.py [hunk 1]
```diff
@@ -984,6 +986,9 @@ def process_raid(self, raid):
 
         gym_info = self.__gym_info.get(gym_id, {})
 
+        #team id saved in self.__gym_hist when processing gym
+        team_id = self.__gym_hist.get(gym_id, -1)
```
Comment (kvangent) [inline PokeAlarm/Manager.py]: This should default to '?' to be consistent with other DTS.

File: PokeAlarm/Manager.py [hunk 2]
```diff
@@ -999,7 +1004,8 @@ def process_raid(self, raid):
             'dir': get_cardinal_dir([lat, lng], self.__location),
             'quick_move': self.__locale.get_move_name(quick_id),
             'charge_move': self.__locale.get_move_name(charge_id),
-            'form': self.__locale.get_form_name(pkmn_id, raid_pkmn['form_id'])
+            'form': self.__locale.get_form_name(pkmn_id, raid_pkmn['form_id']),
+            'team': self.__locale.get_team_name(team_id)
```
Comment (kvangent) [inline PokeAlarm/Manager.py]: Can we change `<team?` to `<team_name>` and then make sure we include `<team_id>` as well? It may be useful for URLs.

**Comment by kvangent** [comment]
Also this will need Wiki support as well.

**Comment by ngovil21** [comment]
Changed as requested.

Wiki update PR here: https://github.com/RocketMap/PokeAlarmWiki/pull/56

**Pull request merged**